### PR TITLE
Feature/twin registry flow

### PR DESCRIPTION
**Pull request by leszekszajnowski-zf**

### CODE REVIEW

**Review by kamiltoszek-zf — APPROVED** [review]

File: explorer/core/src/main/java/net/catenax/explorer/core/ExplorerConfiguration.java [hunk 1]
```diff
@@ -22,38 +25,42 @@
...
-    @Bean
-    ExplorerSearchResultsController explorerSearchResultsController(@Value("${app.use-mockup-data:false}") boolean useMockupData,
-                                                                    ObjectMapper objectMapper,
-                                                                    ExplorerService explorerService,
-                                                                    SpringTemplateEngine templateEngine) {
-        SearchResultsProvider searchResultsProvider = useMockupData ? new MockDataSearchResultsProvider(objectMapper) : new DataSearchResultsProvider(explorerService);
-        return new ExplorerSearchResultsController(searchResultsProvider, templateEngine);
-    }
+  @Bean
+  public LayoutDialect layoutDialect() {
```
Comment (marcingajek-zf) [inline explorer/core/src/main/java/net/catenax/explorer/core/ExplorerConfiguration.java:29]: is LayoutDialect bean needed? I can't see the direct usage

File: explorer/core/src/main/java/net/catenax/explorer/core/aasregistry/controller/ExplorerAasApiDelegate.java [hunk 2]
```diff
@@ -17,17 +17,34 @@
...
-        throw new UnsupportedOperationException("This API endpoint is not yet implemented");
-    }
+  final TwinRegistryService twinRegistryService;
+  final ObjectMapper mapper;
+
+  @Override
+  @SuppressWarnings("unchecked")
+  @SneakyThrows
+  public ResponseEntity<List<AasShellWithMetaDataResponse>> postFetchAasDescriptorsWithMetaData(List<ShellLookup> shellLookup, String bpn) {
+    log.info("Querying for Asset by: " + shellLookup.get(0).getQuery().getAssetIds().get(0).getValue());
```
Comment (marcingajek-zf) [inline explorer/core/src/main/java/net/catenax/explorer/core/aasregistry/controller/ExplorerAasApiDelegate.java:43]: can we be sure that  shellLookup list will not be empty?

File: explorer/core/src/main/java/net/catenax/explorer/core/ExplorerConfiguration.java (hunk 1 above)
Comment (kamiltoszek-zf) [reply explorer/core/src/main/java/net/catenax/explorer/core/ExplorerConfiguration.java:29]: Yes, it's needed and used in thymeleaf templates...

File: explorer/core/src/main/java/net/catenax/explorer/core/aasregistry/controller/ExplorerAasApiDelegate.java (hunk 2 above)
Comment (leszekszajnowski-zf) [reply explorer/core/src/main/java/net/catenax/explorer/core/aasregistry/controller/ExplorerAasApiDelegate.java:43]: no ofc not, I will correct this  soon

**Pull request merged**